src/js/claims-status/containers/AppealInfo.jsx

```diff
@@ -25,6 +24,28 @@ import {
   AVAILABLE,
 } from '../utils/appeals-v2-helpers';
 
+const appealsDownMessage = (
+  <div className="row" id="appealsDownMessage">
+    <div className="small-12 columns">
+      <div className="react-container">
+        <h3>We’re sorry. Something went wrong on our end.</h3>
+        <p>Please refresh this page or try again later. If it still doesn’t work, you can call the Vets.gov Help Desk at <a href="tel:18555747286">1-855-574-7286</a> (TTY: <a href="tel:18008294833">1-800-829-4833</a>). We’re here Monday–Friday, 8:00 a.m.–8:00 p.m. (ET).</p>
```

Comment:
We typically add the + to the href for telephone numbers: href="tel:+1-800-827-1000", we may consider doing so here.